examples/video/main.go

```diff
@@ -19,9 +19,9 @@ func main() {
 	if err != nil {
 		log.Fatalf("could not launch Chromium: %v", err)
 	}
-	page, err := browser.NewPage(playwright.BrowserNewContextOptions{
+	page, err := browser.NewPage(playwright.BrowserNewPageOptions{
 		RecordVideo: &playwright.RecordVideo{
-			Dir: playwright.String("videos/"),
```

Comment:
I in the past used it to differentiate if a user did set a string, since in Go a String can't be nil.

Reply:
According to the documentation, recordVideo itself can be `nil`, and `path` is required once the recordVideo parameter is provided.  When generating type members, the script will judge whether the primitive type needs to be referenced by pointer or not.

```markdown
## context-option-recordvideo
* langs: js
- `recordVideo` <[Object]>
  - `dir` <[path]> Path to the directory to put videos into.
  - `size` ?<[Object]> Optional dimensions of the recorded videos. If not specified the size will be equal to `viewport`
    scaled down to fit into 800x800. If `viewport` is not configured explicitly the video size defaults to 800x450.
    Actual picture of each page will be scaled down if necessary to fit the specified size.
    - `width` <[int]> Video frame width.
    - `height` <[int]> Video frame height.
```